src/rotest/core/flow_component.py

```diff
@@ -231,14 +231,10 @@ def setup_method_wrapper(*args, **kwargs):
                 self.request_resources(self.resources, use_previous=True)
 
             except Exception as err:
-                self.logger.exception("Got an error while getting resources")
-
                 if isinstance(err, ServerError):
                     self.skip_sub_components(self.NO_RESOURCES_MESSAGE)
-                    self.skipTest(self.NO_RESOURCES_MESSAGE)
```

Comment:
Looks like it still skips when resources are not available, no?

Reply:
?
no it doesn't, there's a raise after. I even had to update some ut......